### ISSUE TITLE
(Feature Request) Bring Old Big Picture Back

### ISSUE THREAD

**Issue by dtantono** [ValveSoftware/steam-for-linux#9589]
#### Your system information

* Steam client version (build number or date): 1686779606 (deb)
* Distribution: Ubuntu 22.04
* Opted into Steam client beta?: no
* Have you checked for system updates?: Yes
* Display Server: Xorg
* CPU:  AMD Ryzen 5900 HX
* GPU: Nvidia Geforce RTX 3070
#### Please describe your issue in as much detail as possible:
I know Steam pushed a new update with new user interface. it looks good and consistent with Steam Deck. The problem is  new Steam big picture loads very slow and I think this new design is not ready yet while Stability is most important thing here. So, is it possible to bring old parameter -oldbigpicture back? 

#### Steps for reproducing this issue:

1. turn on Steam application
2.  click Steam Big picture
3. Loads really slow

**Comment by szafranic** [comment]
I would agree, while it is a great UI for the Steam Deck, it is not very good at its original goal of being on a TV. The scaling can be awkward, as well as adding more menu clicks for the same actions. It also cannot compare with the old big picture mode's controller management. Though I would love to see all of that in the new UI as well.

**Comment by blob42** [comment]
Yes please, I didn't use steam since months and today after launching and trying the new UI to check my controller config and it was a horrible experience. I had to check online for tricks to bring back the old UI. If there would be no way to use the old UI I will stop using steam probably as this is the number one feature for me to be able to conveniently use the steam controller.

**Comment by Crackvignoule** [comment]
I just can't play miles Morales on my Steam deck because of the missing setting about Steam input so PLEASE GET BACK THIS

**Comment by joaosagrath** [comment]
Why not leave to the user decide wich one?
keep both UIs.

**Comment by ihatemicrosoft-was-already-taken** [comment]
Apart from everything said above, I think the steam client for Linux is quite buggy. Big picture not starting fullscreen (although configured), transparency issues and more...

So yeah, the option of going back until everything is ruled out would be a big plus!

**Comment by different55** [comment]
Nvidia hardware is still poorly supported on new BPM and has been since it was launched, bringing back the old BPM at least until it's all smoothed over would help a lot.

**Comment by KingMonkeyBug** [comment]
> Why not leave to the user decide wich one? keep both UIs.

I agree!

**Comment by LukasLP13** [comment]
i know this is a old post but has anyone found a way to do it and get it back

**Comment by TeraByte38** [comment]
I would love an option in the big picture modes settings for enabling the old steam pictrue.
Or alternatively an option for the whole steam client to look like it did before the most recent redesign (along with big picture).
